src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -66,8 +66,9 @@ public String getScript() {
         FilePath shf = c.getScriptFile(ws);
 
         String s = script;
-        if (!s.startsWith("#!")) {
-            String defaultShell = Jenkins.getInstance().getInjector().getInstance(Shell.DescriptorImpl.class).getShellOrDefault(ws.getChannel());
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (!s.startsWith("#!") && jenkins != null) {
```

Comment:
Just copy static `getJenkinsInstance()` implementation into some Util class or in this class.